cmd/bios/bios.go

```diff
@@ -27,12 +29,24 @@ func CreateBiosControlCondition(ctx context.Context, action rctypes.BiosControlA
 		return err
 	}
 
-	serverID, err := biosFlags.ParseServerID()
+	serverID, err := uuid.Parse(biosFlags.serverID)
 	if err != nil {
 		return err
 	}
 
-	params := rctypes.NewBiosControlTaskParameters(serverID, action)
+	var biosURL *url.URL
+	if action == rctypes.SetConfig {
+		if biosFlags.biosConfigURL != "" {
```

Comment:
I don't think this if/else is necessary. Cobra should be returning an error if the user fails to set the flag, and the `url.Parse()` will return an error if the user provides an invalid URL.

Reply:
I agree its not necessary. I originally didn't include this check. I do think its worth keeping in the SUPER RARE case someone messes with the code and does something incorrectly so this error will trigger.

Reply:
So now that ive said that, I am willing to go either direction. Whatever you lean with, ill go with.

Reply:
I still feel like it's unnecessary. The concern you have about a code change causing an issue would be better handled by a unit test. So I think we should remove the check, and if we're concerned about a chance causing an issue, then we add a unit test to validate the code's behavior.